repositories.bzl

```diff
@@ -20,23 +20,20 @@ def com_google_gapic_generator_cpp_repositories():
     _maybe(
         http_archive,
         name = "absl",
-        sha256 = "e2b53bfb685f5d4130b84c4f3050c81bf48c497614dc85d91dbd3ed9129bce6d",
         strip_prefix = "abseil-cpp-20181200",
         urls = ["https://github.com/abseil/abseil-cpp/archive/20181200.tar.gz"]
     )
 
     _maybe(
         http_archive,
         name = "com_google_protobuf",
-        sha256 = "73fdad358857e120fd0fa19e071a96e15c0f23bb25f85d3f7009abfd4f264a2a",
-        strip_prefix = "protobuf-3.6.1.3",
-        urls = ["https://github.com/protocolbuffers/protobuf/archive/v3.6.1.3.tar.gz"]
+        strip_prefix = "protobuf-3.7.1",
```

Comment:
Why no sha256?

Reply:
sha256 is hard to maintain, has weird behavior in bazel and often ommited (see grpc repository for example).

The main problem with sha256 is that it if we bump the version of some library but do not bump the sha256 it does not fail normally. One would expect that if sha256 specified and sha256 of the specified new version of the library do not match, there will be an error. But it is not how it works (probably for optimization reasons, since it allows to not try to pull new dependency each time).
Instead bazel just silently keeps using the previous version of the library (i.e. whatever was in the cache on the current system). In simple terms, sha256 is what effectively defines the actual library version, not the download url.

This is super frustrating and is not detectable until somebody cleans their repo or the actual new code expects some change from updated library dependency and fails (because the old, cached, dependency is used). But even in that case, the failure is super confusing (you look at the dependency, see that the new version is specified, but for some reason the old one is pulled, and it takes time to realize that sha256 is the cause).